taxi/views.py

```diff
@@ -59,6 +80,18 @@ class CarListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
     queryset = Car.objects.select_related("manufacturer")
 
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super(CarListView, self).get_context_data(**kwargs)
```

Comment:
does LoginRequiredMixin has this method, so that you should specify from which class you should call this method?